### ISSUE TITLE
It is no longer working

### ISSUE THREAD

**Issue by mikatsume** [robindijkhof/overleaf-textarea#85]
Hi!, I was really excited to find out about your plug-in, but when I downloaded the text window remained empty (I tried changing to Chrome, and I also tried changing the editor version in Overleaf, but nothing). I know you did this out of the goodness of your heart (or something like that) and have no duty to update this, but I wanted to ask if you plan on updating the plug-in.

i hope you have a nice weekend

**Comment by robindijkhof** [comment]
The thing is, Grammarly works out of the box with Overleaf already. Therefore there is no need for this plugin anymore.

**Comment by larsleimbach** [comment]
I don't think so. I often experienced that (with the help of your plug-in) Grammarly finds better suggestions! So, still a wonderful plug-in you created

![image](https://github.com/robindijkhof/overleaf-textarea/assets/57967782/026a29b7-7075-4e21-9512-522b46f04660)
